promscale/sql-api.md

```diff
@@ -17,37 +17,37 @@ WHERE n.nspname OPERATOR(pg_catalog.~) '^(prom)$' COLLATE pg_catalog.default
 ORDER BY 1, 2, 4;
 -->
 
- Name | Arguments | Return type | Description
- --- | --- | --- | ---
+ Name|Arguments|Return type|Description
+  --- | --- | --- | ---
```

Comment:
```suggestion
|-|-|-|-|
```

Reply:
@Rajakavitha1 Check that this table builds as expected - you probably need pipes at the beginning and end of the lines as well.

Reply:
Hi @Loquacity ! I just checked the local build and the table appears fine.
<img width="633" alt="Screenshot 2022-04-12 at 12 01 07 PM" src="https://user-images.githubusercontent.com/37059749/162895418-a2ee7ba8-41cf-4054-b771-a786fd1fab10.png">
<img width="626" alt="Screenshot 2022-04-12 at 12 01 15 PM" src="https://user-images.githubusercontent.com/37059749/162895460-9a3b7a9e-3510-4cfa-b68f-ba680c0634ae.png">